wire-ios-data-model/Tests/Source/ManagedObjectContext/DatabaseMigrationTests.swift

```diff
@@ -138,6 +138,7 @@ final class DatabaseMigrationTests: DatabaseBaseTest {
         }
 
         try allVersions.forEach { version in
+            print("😀 \(version)")
```

Comment:
question: to delete?

Reply:
db2562eb5de21eae7f92be7572dca4a75e614f03